docs/multi-party-computation.md

```diff
@@ -15,7 +15,7 @@ MPC enables computation over inputs from multiple parties while keeping the actu
 
 ### Classic Scenario: The Millionaire Problem
 
-The "Millionaires Problem" is a classic MPC scenario, first introduced by Andrew Yao in 1982. In the "Millionaires Problem" problem, two millionaires want to find out who is richer without disclosing their actual net worth. Using MPC, the millionaires can jointly compute who has more money without revealing their individual net worths to each other or anyone else. This is achieved through a series of cryptographic operations that allow each party to input their net worth into a shared computation. The computation is structured in a way that it only outputs the comparison result (i.e., which millionaire is richer) without leaking any specifics about their respective net worths. This problem showcases the power of MPC - it can preserve privacy while enabling collaborative computation.
+The "millionaires Problem" is a classic MPC scenario, first introduced by Andrew Yao in 1982. In the "millionaires Problem" problem, two millionaires want to find out who is richer without disclosing their actual net worth. Using MPC, the millionaires can jointly compute who has more money without revealing their individual net worths to each other or anyone else. This is achieved through a series of cryptographic operations that allow each party to input their net worth into a shared computation. The computation is structured in a way that it only outputs the comparison result (i.e., which millionaire is richer) without leaking any specifics about their respective net worths. This problem showcases the power of MPC - it can preserve privacy while enabling collaborative computation.
```

Comment:
nit: If this is obvious please ignore it but why do we change M -> m and not P -> p?

Reply:
oops it was a copy paste mistake! updated